packages/css-library/package.json

```diff
@@ -13,11 +13,12 @@
   "scripts": {
     "build:tokens": "style-dictionary build",
     "build:stylesheets": "sass --load-path=./node_modules/@uswds/uswds/packages/ src/stylesheets:dist/stylesheets",
-    "build:minify": "yarn build:stylesheets --style compressed",
-    "build:minify-core": "sass --style compressed dist/core.css:dist/core.min.css",
+    "build:minify-rename": "find dist/stylesheets/min -type f -name '*.css' -exec bash -c 'mv \"$0\" \"${0%.css}.min.css\"' {} \\; && cp -r ./dist/stylesheets/min/* ./dist/stylesheets/ && rm -rf ./dist/stylesheets/min",
```

Comment:
rename all the files to .min.css (sass command line does not have an option to do this)